doc/nrf/releases_and_maturity/migration/migration_guide_2.9.0-nrf54h20-rc1.rst

```diff
@@ -89,6 +89,11 @@ nRF54H20 SoC binaries
 
           nrfutil device x-suit-dfu --firmware nrf54h20_soc_binaries_v0.8.0.zip --serial-number <serial_number>
 
+    #. Purge the device as follows::
```

Comment:
Seems like a duplicate of step no:2. If the user needs to purge again, then could rephrase as below:
```suggestion
    #. Purge the device again as follows::
```